src/middlewared/middlewared/logger.py

```diff
@@ -0,0 +1,137 @@
+import os
```

Comment:
Came to think of it, do we really need all this logger change?

Wouldn't just using
import logging
logging.getLogger('foo')
have the same effect? Since that change was not in stable yet.